.env

```diff
@@ -0,0 +1 @@
+REACT_APP_HELIUS_API= "903d9233-cb1e-4c81-b976-9b1838d2251c"
```

Comment:
remove this from the commit. this should be configured by the user whenever they want to hook up the service locally

Reply:
ah! how embarrassing. 😓

Reply:
I added .env to .gitignore and trashed the helius api key